<ui version="4.0" stdsetdef="1">
  <author>Peter Bartfai</author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ModelTreePanel</class>
  <widget class="QMainWindow" name="ModelTreePanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>446</width>
        <height>362</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>LDView Model Tree</string>
    </property>
    <property name="windowIcon">
      <iconset resource="resources.qrc" >
      <normaloff>:/images/images/LDViewIcon16.png</normaloff>:/images/images/LDViewIcon16.png</iconset>
    </property>
    <widget class="QWidget">
      <layout class="QHBoxLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item>
          <layout class="QVBoxLayout">
            <property name="margin">
              <number>0</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item>
              <widget class="QTreeWidget" name="modelTreeView">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>3</hsizetype>
                    <vsizetype>3</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                </property>
                <column>
                  <property name="text">
                    <string>ModelTree</string>
                  </property>
                  <property name="clickable">
                    <bool>false</bool>
                  </property>
                  <property name="resizable">
                    <bool>false</bool>
                  </property>
                </column>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout">
                <item>
                  <widget class="QCheckBox" name="highlightSelectedLineBox">
                    <property name="text">
                      <string>Highlight selected line</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="highlightColorEdit">
                  </widget>
                </item>
                <item>
                  <spacer name="spacer1">
                    <property name="sizeHint">
                      <size>
                        <width>40</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QPushButton" name="optionsButton">
                    <property name="text">
                      <string>Options &lt;&lt;</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </item>
        <item>
          <widget class="QGroupBox" name="showLinesBox">
            <property name="title">
              <string>Show lines</string>
            </property>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>11</number>
              </property>
              <property name="spacing">
                <number>4</number>
              </property>
              <item>
                <widget class="QCheckBox" name="commentButton">
                  <property name="text">
                    <string>Comment</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="modelButton">
                  <property name="text">
                    <string>Model</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="lineButton">
                  <property name="text">
                    <string>Line</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="triangleButton">
                  <property name="text">
                    <string>Triangle</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="quadButton">
                  <property name="text">
                    <string>Quad</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="conditionalLineButton">
                  <property name="text">
                    <string>Conditional line</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="emptyButton">
                  <property name="text">
                    <string>Empty</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="unknownButton">
                  <property name="text">
                    <string>Unknown</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="Spacer3">
                  <property name="sizeHint">
                    <size>
                      <width>20</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction></pixmapfunction>
  <resources>
   <include location="resources.qrc" />
  </resources>
</ui>
